Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.4.21 (unreleased)
 
 Features:
+ * Code Generator: Make ``molmod(a, b, 0)`` and ``addmod(a, b, 0)`` revert as experimental 0.5.0 feature.
```

Comment:
Need to update all the documentation changes from `revert` to throw (or whatever wording we're using).